### PR TITLE
AboutPage: Update with current text

### PR DESCRIPTION
**Pull request by Olf0**
… from https://github.com/sailfishos-chum/main/blob/main/README.md
Plus use consistent wording and sort issue tracker before source code link.

### CODE REVIEW

**Comment by Olf0** [comment]
I hope to have got the syntax right in the `%//` text sections, especially https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/files#diff-91cbf11fb1958c047c46ddb91ba505b284cbd428455c3ba076cf4ab1f4952918R65-R99
Please check.

**Comment by rinigus** [comment]
Thanks for looking through the messages. That would need some testing, maybe has to wait few days more.

File: qml/pages/AboutPage.qml [hunk 1]
```diff
@@ -27,7 +27,7 @@ Page {
             }
 
             Label {
-                //% "A store frontend for the Chum repository"
+                //% "A graphical application for the SailfishOS:Chum community repository"
                 text: qsTrId("chum-about-store")
                 anchors.horizontalCenter: parent.horizontalCenter
             }
```
Comment (rinigus) [inline qml/pages/AboutPage.qml:33]:
```suggestion
                wrapMode: Text.WordWrap
            }
```

Please add to fit the screen

File: qml/pages/MainPage.qml [hunk 2]
```diff
@@ -60,7 +60,7 @@ Page {
             width: parent.width
 
             PageHeader {
-                title: "SailfishOS:Chum"
+                title: "SailfishOS:Chum GUI"
```
Comment (rinigus) [inline qml/pages/MainPage.qml:63]: I prefer it without GUI and having a shorter form: SailfishOS:Chum. But if you feel strongly about it, can be with GUI as well

File: qml/pages/SettingsPage.qml [hunk 3]
```diff
@@ -48,36 +48,41 @@ Page {
 
                     if (Chum.manualVersion)
                         return Chum.repoTesting ?
-                                    //% "Subscribed to the SailfishOS:Chum testing repository with a manually set Sailfish OS version (%1)"
+                                    //% "Subscribed to the SailfishOS:Chum testing repository with a manually set Sailfish&nbsp;OS version (%1)."
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:51]: &nbsp; doesn't work here, please revert

File: qml/pages/SettingsPage.qml [hunk 4]
```diff
@@ -48,36 +48,41 @@ Page {
...
                                     qsTrId("chum-settings-status-repo-testing-manual").arg(Chum.manualVersion) :
-                                    //% "Subscribed to the regular SailfishOS:Chum repository with a manually set Sailfish OS version (%1)"
+                                    //% "Subscribed to the regular SailfishOS:Chum repository with a manually set Sailfish&nbsp;OS version (%1)."
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:53]: &nbsp; doesn't work here, please revert

File: qml/pages/SettingsPage.qml [hunk 5]
```diff
@@ -48,36 +48,41 @@ Page {
...
                                     qsTrId("chum-settings-status-repo-regular-manual").arg(Chum.manualVersion)
                     return Chum.repoTesting ?
-                                //% "Subscribed to the SailfishOS:Chum testing repository with an automatically determined Sailfish OS version"
+                                //% "Subscribed to the SailfishOS:Chum testing repository with an automatically determined Sailfish&nbsp;OS version."
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:56]: &nbsp; doesn't work here, please revert

File: qml/pages/SettingsPage.qml [hunk 6]
```diff
@@ -48,36 +48,41 @@ Page {
...
                                 qsTrId("chum-settings-status-repo-testing-auto") :
-                                //% "Subscribed to the regular SailfishOS:Chum repository with an automatically determined Sailfish OS version"
+                                //% "Subscribed to the regular SailfishOS:Chum repository with an automatically determined Sailfish&nbsp;OS version."
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:58]: &nbsp; doesn't work here, please revert

File: qml/pages/SettingsPage.qml [hunk 7]
```diff
@@ -103,20 +103,20 @@ Page {
...
                 wrapMode: Text.WordWrap
             }
 
             TextFieldDesc {
                 id: txtRelease
                 //% "Usually a specific SailfishOS:Chum repository is automatically selected according to the "
-                //% "installed Sailfish OS release version. To subscribe to a SailfishOS:Chum repository for "
-                //% "a specific Sailfish OS release, specify this Sailfish OS release here (for example, 4.3.0.12). "
+                //% "installed Sailfish&nbsp;OS release version. To subscribe to a SailfishOS:Chum repository for "
+                //% "a specific Sailfish&nbsp;OS release, specify this Sailfish&nbsp;OS release here (for example, 4.3.0.12). "
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:115]: &nbsp; doesn't work here, please revert

File: qml/pages/SettingsPage.qml [hunk 8]
```diff
@@ -103,20 +103,20 @@ Page {
...
                 text: qsTrId("chum-settings-override-selection")
                 wrapMode: Text.WordWrap
             }
 
             TextFieldDesc {
                 id: txtRelease
                 //% "Usually a specific SailfishOS:Chum repository is automatically selected according to the "
-                //% "installed Sailfish OS release version. To subscribe to a SailfishOS:Chum repository for "
-                //% "a specific Sailfish OS release, specify this Sailfish OS release here (for example, 4.3.0.12). "
+                //% "installed Sailfish&nbsp;OS release version. To subscribe to a SailfishOS:Chum repository for "
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:114]: &nbsp; doesn't work here, please revert

File: qml/pages/SettingsPage.qml [hunk 9]
```diff
@@ -103,20 +103,20 @@ Page {
...
+                //% "a specific Sailfish&nbsp;OS release, specify this Sailfish&nbsp;OS release here (for example, 4.3.0.12). "
                 //% "This is useful when the SailfishOS:Chum repository is not available for the installed "
-                //% "Sailfish OS version, as for Cbeta users."
+                //% "Sailfish&nbsp;OS version, as for Cbeta users."
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:117]: &nbsp; doesn't work here, please revert

File: qml/pages/SettingsPage.qml [hunk 10]
```diff
@@ -103,20 +103,20 @@ Page {
...
                 description: qsTrId("chum-setings-override-release-description")
-                //% "Specify Sailfish OS version"
+                //% "Specify a Sailfish&nbsp;OS version"
```
Comment (rinigus) [inline qml/pages/SettingsPage.qml:119]: &nbsp; doesn't work here, please revert

**Review by rinigus — CHANGES_REQUESTED** [review]
Thank you very much for this improvement. I have tested on XZ2c which is on a smaller side of the screen allowing me to test GUI in what is probably more unforgiving case in terms of line breaks.

Here are the issues I found, please revise

File: qml/pages/AboutPage.qml [hunk 11]
```diff
@@ -15,7 +15,7 @@ Page {
             width: parent.width
 
             PageHeader {
-                //% "About Chum GUI"
+                //% "About the SailfishOS:Chum GUI"
```
[the remark this answers is not in the document]
Comment (Olf0) [reply qml/pages/AboutPage.qml]: Oh, this is an error: I already anticipated that "SailfishOS:Chum GUI application" (here additionally prefixed with "About the ") will likely be too long and thus deleted the "application".  I missed to delete the then superfluous "the ".  Done now per https://github.com/Olf0/sailfishos-chum-gui/commit/df487a3e7766b3db18fb6a2f7c899d6df6eaadf9

If it is still too long, then your suggestion is likely the proper way to go (and I have no idea about the proper syntax for that).

File: qml/pages/SettingsPage.qml (hunk 9 above)
Comment (Olf0) [reply qml/pages/SettingsPage.qml]: Reverted by https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/commits/dfb6335f4c5708b3b86957ce15c5de335344d76e

File: qml/pages/SettingsPage.qml (hunk 8 above)
Comment (Olf0) [reply qml/pages/SettingsPage.qml]: Reverted by https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/commits/dfb6335f4c5708b3b86957ce15c5de335344d76e

File: qml/pages/SettingsPage.qml (hunk 6 above)
Comment (Olf0) [reply qml/pages/SettingsPage.qml]: Reverted by https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/commits/dfb6335f4c5708b3b86957ce15c5de335344d76e

File: qml/pages/SettingsPage.qml (hunk 4 above)
Comment (Olf0) [reply qml/pages/SettingsPage.qml]: Reverted by https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/commits/dfb6335f4c5708b3b86957ce15c5de335344d76e

File: qml/pages/SettingsPage.qml (hunk 5 above)
Comment (Olf0) [reply qml/pages/SettingsPage.qml]: Reverted by https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/commits/dfb6335f4c5708b3b86957ce15c5de335344d76e

File: qml/pages/SettingsPage.qml (hunk 3 above)
Comment (Olf0) [reply qml/pages/SettingsPage.qml]: Reverted by https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/commits/dfb6335f4c5708b3b86957ce15c5de335344d76e

**Comment by Olf0** [comment]
I also reverted the "&nbsp;" in the AboutPage, even though you did not mention them: Was that premature?
I.e., did they work there, although they did not work in the SettingsPage? 

If so, please revert https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/102/commits/5308a2eede3c572440a5a61a640c1936e9d2899a before merging.

**Comment by rinigus** [comment]
> I also reverted the "&nbsp;" in the AboutPage, even though you did not mention them as non-functional: Was that premature?
> I.e., did they work there, although they did not work in the SettingsPage?

It does work in About as it is parsed with HTML subset (`textFormat: Text.StyledText`). It is easier for me to merge first and then revert it. 

As for About page title, I think it will fit in without "the". If not, will make subheader later.

Thank you, merging.

**Comment by Olf0** [comment]
Thank you, this is looking good.

If you want me to take a final look at it on my XperiaX@SFOS3.2.1, notify me when you have uploaded it to the SailfishOS:Chum testing repository.

**Comment by rinigus** [comment]
I have just bumped it in :testing repo, should be ready in 10min or so.

**Comment by rinigus** [comment]
About page is fixed in ca447462e76e461ec0cc3f484788dee8f018e6aa - pushed it directly into the main branch. Corresponding build has been started in :testing, would expect it to be ready soon

**Pull request merged**